loadtest/store/memstore/store.go

```diff
@@ -858,6 +858,19 @@ func (s *MemStore) GetUser(userId string) (model.User, error) {
 	return user, nil
 }
 
+// Users returns all users in the store.
+func (s *MemStore) Users() ([]model.User, error) {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+
+	users := make([]model.User, len(s.users))
+	for _, u := range s.users {
+		users = append(users, *u)
+	}
```

Comment:
If we use append we need to define the capacity, not the length, right?
```suggestion
	users := make([]model.User, 0, len(s.users))
	for _, u := range s.users {
		users = append(users, *u)
	}
```